tests/test_generate_project.py

```diff
@@ -14,18 +15,20 @@
 def base_answers():
     return {
         "full_name": "Jeanne Deau",
-        "email": "jeanne.deau@example.fr",
-        "github_username": "jdeau",
+        "email": "action@github.com",
+        "github_username": "actions-user",
         "project_name": "Snake Farm",
         "project_short_description": "A sample Snake farming project.",
         "version": "0.0.1",
         "open_source_license": "MIT",
         "documentation": True,
         "run_uv_sync": False,
-        "initial_commit": False,
+        "initial_commit": True,
         "setup_github": False,
         "setup_pre_commit": False,
-        "add_me_as_contributor": False,
+        "add_me_as_contributor": platform == "linux",
```

Comment:
Can we leave the base answers as is and test this in a dedicated test case?

Reply:
I removed only “add_me_as_contributor” and created a separate test case which enables it and tests `.allcontributorsc`. Is it enough? What is your concern?